topic-3.4/Roman.java

```diff
@@ -0,0 +1,58 @@
+package bootcamp34.topic3_4;
+
+/**
+ * Class Roman
+ * 
+ * The amounts are added and the exceptions are subtracted 
+ * Special cases: "IV" "IX" "XL" "XC" "CD" "CM"
+ * 
+ * @author Ciel
+ *
+ */
+
+public class Roman {
+	private String roman;
+
+	public Roman(String string) {
+		this.roman = string;
+	}
+
+	public int count(char a) {
+		int count = 0;
+		for (int i = 0; i < roman.length(); i++) {
+			char c = roman.charAt(i);
+			if (c == a) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	public int exception() {
+		int count = 0;
+		char ant = 'w';
+		for (int i = 0; i < roman.length(); i++) {
+			char c = roman.charAt(i);
+			if (i != 0) {
+				ant = roman.charAt(i - 1);
+			}
+			if ((c == 'V' || c == 'X') && (ant == 'I')) {
+				count = count + (1 * 2);
+			} else if ((c == 'L' || c == 'C') && (ant == 'X')) {
+				count = count + (10 * 2);
+			} else if ((c == 'D' || c == 'M') && (ant == 'C')) {
+				count = count + (100 * 2);
+			}
+		}
+		return count;
+	}
+
```

Comment:
[Low] count and exception methods should be declared as private methods. (Also you can rename them as "getCount" and "getException")